src/components/cron.ts

```diff
@@ -0,0 +1,29 @@
+import { getSuggestions, getSuggestionPrintout, updateSuggestionCron } from './suggestions';
+import { TextChannel, MessageEmbed } from 'discord.js';
+import { CommandoClient } from 'discord.js-commando';
+import { CronJob } from 'cron';
+import { EMBED_COLOUR } from '../utils/embeds';
+
+const MOD_CHANNEL_ID: string = process.env.MOD_CHANNEL_ID || '.';
+
+// Checks for new suggestions every min
+export const createSuggestionCron = (client: CommandoClient): CronJob =>
+  new CronJob('0 */1 * * * *', async function () {
+    const createdSuggestions = await getSuggestions('created');
```

Comment:
I don't really like having just string literals here for our states, maybe create an enum of states? Kinda like what we had before but we can use string enums this time.

```
enum SuggestionState {
  Created = 'created'
  Pending = 'pending'
  ...
}
```

Reply:
you can rename the current `suggestionStates` object to `readableSuggestionStates`